pkgs/games/xonotic/default.nix

```diff
@@ -61,18 +61,9 @@ let
 
     src = fetchurl {
       url = "https://dl.xonotic.org/xonotic-${version}-source.zip";
-      sha256 = "0axxw04fyz6jlfqd0kp7hdrqa0li31sx1pbipf2j5qp9wvqicsay";
+      sha256 = "sha256-oagbpVqxUb8JdY5/WWFLLlFQ6EIkdT53lQvNB6KC6l0=";
     };
 
-    patches = [
-      # Fix to make darkplaces compile under GCC 11
-      (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
good catch, dropped